plugin/trino-kafka/src/test/resources/protobuf/google/protobuf/timestamp.proto

```diff
@@ -0,0 +1,138 @@
+// Protocol Buffers - Google's data interchange format
```

Comment:
Shouldn't we refer them from `wire-schema` or `protobuf-util` jar ?

Reply:
Unfortunately that's file is need to be included if imported in some proto file(s), to Kafka's serializer be working on send (see error reproduction [details](https://github.com/trinodb/trino/pull/15312/commits/5b4932d1031ed8cb39b12500a29ea41640d36784#diff-e59ff6a36068bd7da2bc7a4c4996a696ddac09a449542a981673cafee5643172R225))

Reply:
But the file is already shipped in `protobuf-java` library

Reply:
I have tested it locally by removing the file and it test ran successfully. Is there is any way to configure the test ?

Reply:
re-tested it too, you're right. thank you for the finding! removed from PR.